crates/swc_ecma_minifier/tests/fixture/issues/issue-2604/1/output.js

```diff
@@ -1,16 +1,18 @@
 push({
     "": function() {
-        function za() {
-            try {
-                return eval(a);
-            } catch (b) {}
-        }
-        function Rb(a1) {
-            return p() ? JSON : za();
-        }
-        h.get = function(a1) {
-            return null == a1 ? null : Rb(a1);
-        };
+        !function() {
```

Comment:
Don't know why but eval prevents inlining

Reply:
It was intentional as we should preserve scope around `eval`